pjsip/src/pjsua-lib/pjsua_aud.c

```diff
@@ -479,6 +479,9 @@ pj_status_t pjsua_aud_subsys_destroy()
 
     /* Destroy file players */
     for (i=0; i<PJ_ARRAY_SIZE(pjsua_var.player); ++i) {
+	if (pjsua_var.player[i].pool) {
+	    PJ_LOG(2,(THIS_FILE, "Pool for player id=%d is not released"));
+	}
```

Comment:
Perhaps the warning message is better about "destructor not called" instead of "pool not released" (can be put in the same "if" block below)?